### ISSUE TITLE
Setup: Enhancements

### ISSUE THREAD

**Issue by muratpurc** [CONTENIDO/CONTENIDO#484]
Optimize following parts in setup:

- Display the proper CONTENIDO version in message “`You are updating a previous version of CONTENIDO to...`".
- Do table checks only for CONTENIDO related tables (tables having the prefix).

See related comment in forum:
https://forum.contenido.org/viewtopic.php?f=115&t=44755&p=183450#p183441

**Comment by dceonline** [comment]
@muratpurc did somebody has yet tested to upgrade versions prior 4.9 directly to 4.10 or the develop branch (4.11.0)?

Maybe we have to rework this part of setup, function doChangedDirsFilesTest(), showing only a message to upgrade older versions step by step, starting with version 4.9 or another knwon working version.

Meanwhile i fixed this table check stuff and added it to #630

**Comment by muratpurc** [comment]
@dceonline Yes, I tested the update from CONTENIDO 4.8 to the development branch (as of February 2026), and a problem occurred in certain cases. Unfortunately, I haven't yet had the chance to narrow down the issue further or work on it.
Therefore, I do not recommend updating from CONTENIDO 4.6 or 4.8 to the latest version.
